libopenage/renderer/stages/camera/manager.cpp

```diff
@@ -15,7 +15,8 @@ CameraManager::CameraManager(const std::shared_ptr<renderer::camera::Camera> &ca
 	move_motion_directions{static_cast<int>(MoveDirection::NONE)},
 	zoom_motion_direction{static_cast<int>(ZoomDirection::NONE)},
 	move_motion_speed{0.2f},
-	zoom_motion_speed{0.05f} {
+	zoom_motion_speed{0.05f},
+	camera_boundaries({X_MIN, X_MAX, Z_MIN, Z_MAX}) {
```

Comment:
You can remove the outer parentheses for initialization.

```suggestion
	camera_boundaries{X_MIN, X_MAX, Z_MIN, Z_MAX} {
```